src/form_helper.js

```diff
@@ -98,37 +102,33 @@ export default class FormHelper {
   // check all of the elements in the formHelper and confirm they are still
   // within the page DOM
   areAllElementsStillInTheDOM(){
-    if(!document.body.contains(this.formHelperConfig.countryElement)){
+    const doesntContainElement = element => !document.body.contains(element)
+
+    if( doesntContainElement(this.formHelperConfig.countryElement)){
       this._log("Country Element is not in the DOM")
       return false
     }
 
-    // TODO can we aggregate the elements to check into a single array or map?
-
     const countryCodes = ['nz', 'au']
```

Comment:
how about defining `country_codes` as a class `const` as it's used in more than one place.